server/plugin/utils.go

```diff
@@ -384,3 +386,22 @@ func lastN(s string, n int) string {
 
 	return string(out)
 }
+
+func getOrganizations(organizations string) ([]string, map[string]bool) {
```

Comment:
If we make this a method of `Configuration`, there is no need to pass in the argument,  which seems to simplify the code.